packages/blade/rollup.config.js

```diff
@@ -42,6 +47,15 @@ const inputRootDirectory = 'src';
 const outputRootDirectory = 'build';
 const exportCategories = ['components', 'tokens', 'utils'];
 
+const aliases = pluginAlias({
```

Comment:
doing all these things in rollup is so straightforward - add a plugin, define rules done vs doing it in webpack 😭

Reply:
Yeah rollup is nicer.